scripts/get_tlabel.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+PR="${1}"
+
+>&2 printf $'Using PR: \'%s\'\n' "${PR}"
+
+tlabels="$(gh api --jq '.labels.[].name' "${PR}" | grep -- '^t-' || printf 'generic')"
+# print to error, since the stdout is captured into `GITHUB_OUTPUT
+>&2 printf 't-labels:\n---\n%s\n---\n' "${tlabels}"
+# if there isn't exactly 1 `t-xxx` label, use `generic`
+if [[ "$(wc -l <<<"${tlabels}")" -ne 1 ]]; then
+  tlabels="generic"
+fi
+streamName="${tlabels} maintainer merges"
+>&2 printf $'Post to topic: \'%s\'"\n' "${streamName}"
+echo "topic=${streamName}"
```

Comment:
In hindsight, I think I prefer all the topics starting with `maintainer merge` for visual organization/alignment purposes. But that's a minor concern.
```suggestion
topicName="$maintainer merge: {tlabels}"
>&2 printf $'Post to topic: \'%s\'"\n' "${topicName}"
echo "topic=${topicName}"
```